lib/elasticsearch_models/base.rb

```diff
@@ -36,6 +37,17 @@ def build!(*params)
         model
       end
 
+      def save_model!(model)
```

Comment:
Is this method intended to be public?

Reply:
It is called by an instance, so it needs to be.  Rails public / private access is very weird....

Reply:
But it looks like the instance is the only thing calling it, so it could be a private instance method. Given that it doesn't have test coverage and is not in the README, it'd be best that it not be accessible to consumers of the gem.